### PR TITLE
Feature/get confluence page

### PR DESCRIPTION
**Pull request by Horsty**

### CODE REVIEW

File: swagger-confluence-client.yaml [hunk 1]
```diff
@@ -44,6 +44,33 @@ paths:
           description: "Returned if there is no space with the given key, or if the calling user does not have permission to view the space."
           schema:
             $ref: '#/definitions/confluenceSpaceError'
+  /content/{pageId}:
+    get:
+      summary: "Gets a confluence page content by id."
+      operationId: "getPage"
```
Comment (schorsch3000) [inline swagger-confluence-client.yaml]: Shouldn't that be getPageById?

File: swagger-confluence-client.yaml [hunk 2]
```diff
@@ -44,6 +44,33 @@ paths:
...
+          required: true
+      consumes:
+        - "application/json"
+      produces:
+        - "application/json"
+      responses:
+        200:
+          description: "Returns a full JSON representation of a piece of content"
+          schema:
+            $ref: '#/definitions/confluencePageExists'
```
Comment (schorsch3000) [inline swagger-confluence-client.yaml]: ...Exists implies it's a check and will return a boolean, this will return a full page.

**Review by schorsch3000 — CHANGES_REQUESTED** [review]

File: swagger-confluence-client.yaml (hunk 1 above)
Comment (Horsty) [reply swagger-confluence-client.yaml]: You're right. That's more specific. I'll get right to it and change it.

File: swagger-confluence-client.yaml (hunk 2 above)
Comment (Horsty) [reply swagger-confluence-client.yaml]: Aye. I rename it into confluencePageRepresentation.

File: swagger-confluence-client.yaml [hunk 3]
```diff
@@ -47,7 +47,7 @@ paths:
   /content/{pageId}:
     get:
       summary: "Gets a confluence page content by id."
-      operationId: "getPage"
+      operationId: "getPageById"
```
Comment (dithmer) [inline swagger-confluence-client.yaml]: We tried to use the Atlassian-API-documentation names as operationId. So it should be "getContentById" 
https://docs.atlassian.com/ConfluenceServer/rest/7.0.4/#api/content-getContentById

File: swagger-confluence-client.yaml [hunk 4]
```diff
@@ -44,6 +44,33 @@ paths:
...
+  /content/{pageId}:
+    get:
+      summary: "Gets a confluence page content by id."
+      operationId: "getPage"
+      tags:
+        - confluence
+      description: "Returns a piece of Content."
+      security:
+        - crowdAuth: []
+      parameters:
```
Comment (dithmer) [inline swagger-confluence-client.yaml]: Missing query parameters for status, version, expand

https://docs.atlassian.com/ConfluenceServer/rest/7.0.4/#api/content-getContentById

**Review by dithmer — CHANGES_REQUESTED** [review]

File: swagger-confluence-client.yaml (hunk 3 above)
Comment (Horsty) [reply swagger-confluence-client.yaml]: Okay. I'll change it

File: swagger-confluence-client.yaml (hunk 4 above)
Comment (Horsty) [reply swagger-confluence-client.yaml]: I'll add them in the next commit

**Pull request merged**